src/components/Combobox/components/ComboboxDropdown.js

```diff
@@ -55,6 +58,32 @@ const ComboboxDropdown = React.forwardRef((props, ref) => {
     dropdownClassName,
   ];
 
+  const comboboxEl = (
+    <Fragment>
+      <div className={cx(comboboxContainerClasses)}>
+        <div
+          className={cx(comboboxClasses)}
+          aria-expanded={isOpen}
+          aria-haspopup="listbox"
+          role="combobox"
+        >
+          <div className={cx(comboboxFormElementClasses)} role="none">
+            {renderInput()}
+          </div>
+          <div
+            className={cx(dropdownClasses)}
+            id={listboxId}
+            ref={ref}
+            role="listbox"
+            tabIndex="-1"
+          >
+            {children}
+          </div>
+        </div>
+      </div>
+    </Fragment>
```

Comment:
Why do we need `Fragment` here?

Reply:
Good find. We actually don't, it's a leftover from an earlier iteration. Will remove